cmd/project.go

```diff
@@ -65,13 +67,59 @@ func (p *Project) Create() error {
 	}
 
 	// create license
-	return p.createLicenseFile()
+	if p.Legal.Name != "None" {
+		return p.createLicenseFile()
+	}
+	return nil
 }
 
 func (p *Project) createLicenseFile() error {
 	data := map[string]interface{}{
 		"copyright": copyrightLine(),
 	}
+	licensesExist := []string{}
+	err := filepath.Walk(p.AbsolutePath, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+		if info.IsDir() && filepath.Ext(path) != ".txt" && filepath.Ext(path) != ".md" && filepath.Ext(path) != "" {
+			return nil
+		}
+		reg := regexp.MustCompile(`(?i).*license\.?.*`)
```

Comment:
Used Regex to find License in the project directory.
Regex: _Should contain the word license and should not have an extension / should be a .txt or .md_